cmd/osctl/cmd/kubeconfig.go

```diff
@@ -25,7 +25,7 @@ var kubeconfigCmd = &cobra.Command{
 	Short: "Download the admin kubeconfig from the node",
 	Long: `Download the admin kubeconfig from the node.
 Kubeconfig will be written to PWD/kubeconfig or [local-path]/kubeconfig if specified.`,
-	Args: cobra.ExactArgs(1),
+	Args: cobra.MaximumNArgs(1),
```

Comment:
bug I introduced in previous PR, as now it requires an arg, while zero args should work (now with tests!)